### ISSUE TITLE
[nuka-carousel] Error while generating js file with fuse-box

### ISSUE THREAD

**Issue by SyrineBen** [fuse-box/fuse-box#1955]
I am using fuse-box@4.0.0-next.444 with nuka-carousel@4.6.6, this dependency uses d3-ease@1.0.7, when I generate js file for my app, I have this error `Uncaught SyntaxError: Invalid left-hand side expression in prefix operation`
The line that contains the error in my js file : 
```
 function elasticIn(t) {
    return a * math_js_1.tpmt(---t) * Math.sin((s - t) / p);
  }
```
When I look to this function from the node module d3-ease, I find : 
```
function elasticIn(t) {
    return a * tpmt(-(--t)) * Math.sin((s - t) / p);
  }
```
The bug is somehow fusebox compiles -(--t) into ---t .
Can you solve this bug please ?
Thank you in advance

**Comment by starsolaris** [comment]
Is this fuse-box issue or some of fuse-box dependencies?

**Comment by Gouigouix** [comment]
Hello @starsolaris, @SyrineBen,

It is indeed the lib "d3-ease" which made a modification in its code that the TypeScript compiler interprets in order to optimize the code by removing the parentheses. There is no error or warning during compilation, however the browser crashes at runtime!

I requested a change from the lib dev which you can see here https://github.com/d3/d3-ease/commit/7d24c5f6d73f783acee85feb9293b57e8fa3c8d4#r47318271

Thank you

**Comment by Gouigouix** [comment]
> Is this fuse-box issue or some of fuse-box dependencies?

No connection with FuseBox, you can close the issue 😉

**Comment by charsleysa** [comment]
This is not an issue with typescript, this is an issue with fuse-box.

**Comment by Gouigouix** [comment]
> This is not an issue with typescript, this is an issue with fuse-box.

Indeed, I just tested I do not have this problem when I use compilation by tsc. I thought that Fuse-box had done no processing on the code but had only created a wrapper around the TypeScript compilation!
